src/engine/renderer/tr_init.cpp

```diff
@@ -162,9 +162,12 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 	cvar_t      *r_specularExponentMin;
 	cvar_t      *r_specularExponentMax;
 	cvar_t      *r_specularScale;
+	cvar_t      *r_specularMapping;
 	cvar_t      *r_deluxeMapping;
 	cvar_t      *r_normalScale;
 	cvar_t      *r_normalMapping;
+	cvar_t      *r_parallaxDepthScale;
```

Comment:
These seem to be in the wrong commit but that's OK, it's not going to hurt anyone.

Reply:
OW! Stubbed my toe on this.

Reply:
this just moves stuff along similar things for better reading

Reply:
That probably slipped up with some fixup, that rebase switching to `USE_` to `r_` macros is the most crazy one I ever did, with a file split in the middle.

Reply:
I extracted it to its own commit